src/shared/lib/mutationUtils.ts

```diff
@@ -0,0 +1,5 @@
+import {GENETIC_PROFILE_UNCALLED_MUTATIONS_SUFFIX} from "../constants";
+
+export function isUncalled<T extends {geneticProfileId?:string}>(mutation:T) {
+    return Boolean(mutation.geneticProfileId && mutation.geneticProfileId.match(new RegExp(GENETIC_PROFILE_UNCALLED_MUTATIONS_SUFFIX + "$")));
```

Comment:
this should be new RegExp(...).test(mutation.geneticProfileId).  the test method of RegExp returns boolean and handles undefined arg.